boards/arc/iotdk/doc/board.rst

```diff
@@ -0,0 +1,200 @@
+   :goals: menuconfig
+
+
+Building
+--------
+
+You can build application in the usual way.  Refer to
+:ref:`build_an_application` for more details. Here is an example for
+:ref:`hello_world`.
+
+.. zephyr-app-commands::
+   :board: iotdk
+   :zephyr-app: samples/hello_world
+   :maybe-skip-config:
+   :goals: build
+
+Connecting Serial Output
+=========================
+
+In the default configuration, Zephyr's IoT Development Kit images support
+serial output via the USB-UART on the board.  To enable serial output:
+
+On your development environment, you will need to:
+
+* Open a serial port emulator (i.e. on Linux minicom, putty, screen, etc)
+* Specify the tty driver name, for example, on Linux this may be :file:`/dev/ttyUSB0`
+* Set the communication settings to:
+
+
+========= =====
+Parameter Value
+========= =====
+Baud:     115200
+Data:     8 bits
+Parity:    None
+Stopbits:  1
+========= =====
+
+Debugging
+==========
+
+Using the latest version of Zephyr SDK(>=0.9), you can debug and flash
+IoT Development Kit directly.
+
+One option is to build and debug the application using the usual
+Zephyr build system commands.
+
+.. zephyr-app-commands::
+   :board: iotdk
+   :app: <my app>
+   :goals: debug
+
+At this point you can do your normal debug session. Set breakpoints and then
+'c' to continue into the program.
+
+The other option is to launch a debug server, as follows.
+
+.. zephyr-app-commands::
+   :board: iotdk
+   :app: <my app>
+   :goals: debugserver
+
+Then connect to the debug server at the IoT Development Kit from a second
+console, from the build directory containing the output :file:`zephyr.elf`.
+
+.. code-block:: console
+
+   $ cd <my app>
+   $ $ZEPHYR_SDK_INSTALL_DIR/sysroots/x86_64-pokysdk-linux/usr/bin/arc-zephyr-elf/arc-zephyr-elf-gdb \
+      zephyr.elf
+   (gdb) target remote localhost:3333
+   (gdb) load
+   (gdb) b main
+   (gdb) c
+
+Flashing
+========
+
+If you just want to download the application to the IoT Development Kit's CCM
+and run, you can do so in the usual way.
+
+.. zephyr-app-commands::
+   :board: iotdk
+   :goals: flash
+
+This command still uses openocd and gdb to load application elf file
+to IoT Development Kit, but it will load application and then run immediately.
+If power is lost, the application will also lost due to power loss.
+
+Most of the time you will not be flashing your program but will instead
+debug it using openocd and gdb. The program can be download via the USB
+cable into the code and data memories.
+
+When you are ready to deploy the program so that it boots up automatically
+on reset or power-up, you can follow the steps to place the program on
+SPI-FLASH or interal eFlash.
+
+For instructions on how to write your program to FLASH,
+refer to the documentation on the IoT Development Kit at the
+`embARC website`_.
+
+Release Notes
+*************
+
+The following is a list of TODO items:
+
+* :jira:`GH-2647`: Zephyr needs i-cache API (all targets)
```

Comment:
The Zephyr project isn't using Jira for tracking issues.  What is this referring to?

Reply:
removed